Makefile

```diff
@@ -36,7 +36,7 @@ endif
 gofmt:
 ifneq ($(HAS_SERVER),)
 	@echo Running gofmt
-	@for package in $$(go list ./server/...); do \
```

Comment:
Hmm, do we need to drop the `server` prefix here? This only gets run if `HAS_SERVER`, so it won't help catch linting issues say, in the manifest tool, if it's a webapp only plugin anyway. 0/5, since it probably doesn't hurt.

Reply:
Good catch, that this only get run if `HAS_SERVER`. My idea was that this was checking the files in `build` "for us" and catch issues when we change these files.